src/main/java/com/keemerz/klaverjas/repository/CredentialsRepository.java

```diff
@@ -1,18 +1,15 @@
 package com.keemerz.klaverjas.repository;
 
 import com.keemerz.klaverjas.domain.Credentials;
-import com.keemerz.klaverjas.domain.Player;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.stereotype.Component;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.UUID;
 
 @Component
 public class CredentialsRepository {
 
-    private static final CredentialsRepository INSTANCE = new CredentialsRepository();
```

Comment:
I would expect calling code to be modified as well. Or was this unused?

Reply:
I am... not sure.

Reply:
Apparently that was already merged in the other PR